mapsims/noise.py

```diff
@@ -85,7 +86,11 @@ def __init__(
             If True, reduce the hitcount by a factor of 0.85 to account for not-uniformity in the scanning
         scanning_strategy : str
             Choose between the available scanning strategy hitmaps "classical" or "opportunistic" or
-            path to a custom hitmap; it will be normalized, so absolute hitcount does not matter
+            path to a custom hitmap; it will be normalized, so absolute hitcount does not matter.
+            If None, or "homogenous" or "isotropic", then no hitmap scaling is applied and 
+            a homogenous noise field is generated on the full sky. In this case, the sky fraction fsky must be
+            specified, which will be used to populate the full sky with homogenous noise with noise power
+            equal to what SO would obtain if it observed only a partial sky of fraction fsky.
```

Comment:
I have modified this section also in my other standing pull request. We will have to merge the 2 changesets somehow